x/tokenfactory/keeper/admins_test.go

```diff
@@ -8,6 +8,32 @@ import (
 	"github.com/sei-protocol/sei-chain/x/tokenfactory/types"
 )
 
+func (suite *KeeperTestSuite) TestBuggyScenario() {
```

Comment:
Can we rename this to something more explicit around the bug?

Reply:
yeah sorry, will do, had just named it while trying to repro the issue and did it hastily